.github/ISSUE_TEMPLATE/New Issue.yml

```diff
@@ -2,10 +2,6 @@ name: New Issue
 description: List an issue for the improvement of WAM2layers
 body:
   - type: markdown
-    attributes:
-      description: |
-        If you have a question, idea or want to engage with the community, check 'Discussions' and the flowchart in the readme first. 
-        If you want to contribute to the development and know what you are doing, you are welcome to open a new issue. 
-        In any case please introduce yourself if you are a new user
+    value: "If you have a question, idea or want to engage with the community, check 'Discussions' and the flowchart in the readme first. If you want to contribute to the development and know what you are doing, you are welcome to open a new issue. In any case please introduce yourself if you are a new user."
```

Comment:
Would be nice if "discussions" was a hyperlink to the discussion page.

Reply:
What about: 

Welcome to WAM2layers! 

If you're new here, please check our [discussion forum](https://github.com/WAM2layers/WAM2layers/discussions) first. The discussion forum is a good place to ask questions, discuss ideas, or get to know the community.